plugins/module_utils/network/sonic/argspec/lst/lst.py

```diff
@@ -0,0 +1,80 @@
+#
+# -*- coding: utf-8 -*-
+# Copyright 2024 Dell Inc. or its subsidiaries. All Rights Reserved.
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+#############################################
+#                WARNING                    #
+#############################################
+#
+# This file is auto generated by the resource
+#   module builder playbook.
+#
+# Do not edit this file manually.
+#
+# Changes to this file will be over written
+#   by the resource module builder.
+#
+# Changes should be made in the model used to
+#   generate this file or in the resource module
+#   builder template.
+#
+#############################################
+
+"""
+The arg spec for the sonic_lst module
+"""
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+class LstArgs(object):  # pylint: disable=R0903
+    """The arg spec for the sonic_lst module
```

Comment:
Triple quotes for a docstring should have their own line. Descriptions following the triple quotes should not be on one line I believe.
```suggestion
    """
    The arg spec for the sonic_lst module
```